st/m0d-device-detach-test.sh

```diff
@@ -68,7 +68,7 @@ M0_SPIEL_OPTS="-l $M0_SRC_DIR/motr/.libs/libmotr.so --client $SPIEL_ENDPOINT \
                --ha $HA_ENDPOINT"
 
 function spiel_cmd {
-    $M0_SRC_DIR/utils/spiel/m0spiel $M0_SPIEL_OPTS <<EOF
+    "$M0_SRC_DIR"/utils/spiel/m0spiel "$M0_SPIEL_OPTS" <<EOF
```

Comment:
I have doubt on this: "$M0_SPIEL_OPTS"

$M0_SPIEL_OPTS is a list of arguments to be passed to the command 'm0spiel'
If it is quoted, the list of arguments will be passed to 'm0spiel' as a single argument.
Please refer to:
https://github.com/Seagate/cortx-motr/pull/1905
https://github.com/Seagate/cortx-motr/pull/1730

We need to find a way to differentiate these use cases.

Reply:
Please let me know if I should remove this particular double quote reference.

Reply:
Yes, because it is wrong to do so.